nautobot/ipam/models.py

```diff
@@ -499,7 +499,7 @@ class Prefix(PrimaryModel, StatusModel):
 
     class Meta:
         ordering = (
-            F("vrf__name").asc(nulls_first=True),
+            F("vrf_id").asc(nulls_first=True),
```

Comment:
Using `"vrf__name"` means we have to join against the `ipam_vrf` table to apply the ordering. This both hurts performance and is incompatible with `.select_for_update()` because the VRF field is allowed to be `NULL`. On Postgres this triggers the error: `ERROR: FOR UPDATE cannot be applied to the nullable side of an outer join`.

Ordering by `vrf_id` means that Prefix results will still be grouped by VRF, they just won't be in alphabetic order.